source/renderer/app/components/status/formatCpuInfo.spec.js

```diff
@@ -0,0 +1,29 @@
+import formatCpuInfo from './formatCpuInfo';
+
+describe('Formatters/formatCpuInfo', () => {
+  it('gives correct model name and clock speed for Intel format', () => {
+    expect(
```

Comment:
Do you think it would be beneficial to add a test case for edge cases eg. when one of the params is missing?